sharepoint/sharepoint-ps/sharepoint-online/Connect-SPOService.md

```diff
@@ -176,6 +176,32 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### -ModernAuth
+
+ Ensures that SharePoint Online tenant administration cmdlets can connect to the service using modern TLS protocols.
+
+To use it you also need to provide the **AuthenticationUrl** parameter.
+
+Example:
+
+$creds = Get-Credential
+
+ ```powershell
+Connect-SPOService -Credential $creds -Url https://tenant-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations
```

Comment:
I tested this example and it only worked if I removed the -Credential, can you please check? Thanks

Reply:
Confirmed on multiple machines this works.

Did you ever get a prompt when you ran $creds = Get-Credential? If not , you may need to run PowerShell in administrator mode.

Reply:
Yes, I got a basic auth prompt, with a non-MFA account it worked, but with a MFA enabled account it failed, maybe I am supposed to do something different? Thanks for clarifying

Reply:
Ya sorry for MFA it would be minus credential. If we need to move this section to be clearer, that's fine.

Reply:
Yes, can you do it? We do not have permissions to edit the PR, thanks